app/views/landlineNumber/CaptureLandlineNumberView.scala.html

```diff
@@ -24,9 +24,9 @@
 @(landlineForm: Form[String], currentLandline: String)(implicit user: User[_], messages: Messages, appConfig: AppConfig)
 
 @questionContent = {
-    <h1 id="page-heading" class="heading-large">@messages("captureLandline.title")</h1>
-
-    <span class=form-hint>@messages("captureLandline.hint")</span>
+    <h1 id="page-heading">
+        <label id="heading-text" class="heading-large">@messages("captureLandline.title")</label>
```

Comment:
Might need to do this a little bit differently and include what the label is **for** 

ie. (I tried to paste in the code but git was clever enough to decode the html!)
https://design-system.service.gov.uk/components/text-input/